index.js

```diff
@@ -1,11 +1,17 @@
-import {runAppleScript} from 'run-applescript';
+import { runAppleScript } from "run-applescript";
 
 /**
  * @param {string} profile
  * @return {Promise<void>}
  */
 export async function setTerminalProfile(profile) {
-	await runAppleScript(`tell application "Terminal"
+  await runAppleScript(`tell application "Terminal"
```

Comment:
The formatting changes were applied automatically by my editor via Prettier. I figured it wouldn't hurt to leave them in.

Reply:
this project uses [XO](https://github.com/xojs/xo)

https://github.com/patrik-csak/terminal-profile/blob/54dd4d4d38667b2cbd4ae725747a78752ee27b7d/package.json#L18